tests/hca/test_submission.py

```diff
@@ -0,0 +1,90 @@
+import unittest
+
+from hca.submission import HcaSubmission
+from tests.hca.utils import make_ingest_entity, random_id, random_uuid
+
+
+class TestHcaSubmission(unittest.TestCase):
+    def test_added_entity_is_contained_in_hca_submission(self):
```

Comment:
We could define a setUp method for set the `test_id` and `test_uuid` fields.

```suggestion

def setUp:
    self.test_id = random_id()
    self.test_uuid = random_uuid()

def test_added_entity_is_contained_in_hca_submission(self):
`

Then you would have to replace the  `test_id` and `test_uuid` with  `self.test_id` and `self.test_uuid` and remove the unnecessary lines.